api/src/main/java/org/openmrs/Obs.java

```diff
@@ -1080,11 +1080,31 @@ public String getValueAsString(Locale locale) {
 		return "";
 	}
 	
-	private static DateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd");
+	private static ThreadLocal<DateFormat> dateFormat = new ThreadLocal<DateFormat>() { // Made DateFormat Threadsafe using ThreadLocal		

```

Comment:
Is this better than just replacing the static dateFormat, timeFormat, and datetimeFormat fields with calls to: new SimpleDateFormat(...)? E.g. is introducing more performant than just instantiating these as-needed? (e.g. OpenmrsUtil.getDateFormat methods instantiate new ones...)